js/app.js

```diff
@@ -0,0 +1,274 @@
+/*
+
+// 1. boolean - there are only true states
+// 2. string - does it contain @?
+// 3. Objects - there are a lot of working parts to one thing
+// 4. Array - A lot of things with no detail
+// 5. Objects - there are a lot of working parts to one thing, with more detail.
+// 6. Objects - there are a lot of working parts to one thing, with even more detail.
```

Comment:
**2/6** your answers are perfect but the assignment wanted you to actually create the things